test/routes/bespokeApplyOffline.route.test.js

```diff
@@ -18,9 +18,15 @@ const Routes = require('../../src/routes')
 
 const routesToTest = {}
 
+// Iterate over Routes and add any routes with the controller bespokeApplyOffline to the routesToTest object. We have a
+// deny list of routes which have this controller but which do not have behaviour defined in
+// BespokeApplyOfflineController. These routes should be removed from the deny list if and when the controller is
+// updated to act on them.
+const routeDenyList = ['FACILITY_APPLY_OFFLINE', 'WASTE_ASSESSMENT_APPLY_OFFLINE']
```

Comment:
This means that these two routes aren't being tested.

I think the correct thing to do is probably amend the tests so that they don't check for the activity types on these two routes.